ui/src/scss/_list.scss

```diff
@@ -113,9 +113,8 @@
     padding-top: 0;
     .page-middle {
         .list-head {
+            margin-top: 15px;
```

Comment:
I'm not sure what this is for?

Reply:
it's mainly just to align the list head with other elements such as the list summary, as the rest of the heads across other pages never touch the header and also go with a similar layout.

Reply:
Actually, some of the other pages also do touch the header, e.g. the profile page.

Reply:
ah, I'll just yoink that margin out then

Reply:
Thanks.